relay/nakama/handlers.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	errors2 "errors"
+	"github.com/heroiclabs/nakama-common/runtime"
+	"github.com/rotisserie/eris"
+	"io"
+	"net/http"
+	"pkg.world.dev/world-engine/relay/nakama/allowlist"
+	"pkg.world.dev/world-engine/relay/nakama/errors"
```

Comment:
nit: the standard "errors" package should keep its name, and this should be nakamaerrors.